src/main/resources/application-local.yaml

```diff
@@ -1,14 +1,22 @@
 app:
   base-url: http://localhost:8080
 
-api:
-  apiKeys:
-    - keyvalue: foobar
+tenant:
```

Comment:
added this as a todo in my comment.

Reply:
changing the structure of tenants can be challenging, but if you want to change this naming, we need to be very careful since you need to ensure the old code with old properties still works.